src/test/scala/de/zalando/beard/renderer/BugSpec.scala

```diff
@@ -1,39 +0,0 @@
-package de.zalando.beard.renderer
-
-import com.typesafe.scalalogging.LazyLogging
-import org.scalatest.{Matchers, FunSpec}
-
-import scala.io.Source
-
-/**
-  * @author dpersa
-  */
-class BugSpec extends FunSpec with Matchers with LazyLogging {
```

Comment:
But I added the test that checks this case into the BeardTemplateParserSpec.scala. Check please:

https://github.com/kaygorodov/beard/blob/a059681bf23d5977b0e3b3ae8d352f5a9217d1fa/src/test/scala/de/zalando/beard/parser/BeardTemplateParserSpec.scala#L195